src/connectors/impls/gcl/writer.rs

```diff
@@ -0,0 +1,239 @@
+// Copyright 2022, The Tremor Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub(crate) mod meta;
+mod sink;
+
+use crate::connectors::impls::gcl::writer::sink::GclSink;
+use crate::connectors::prelude::*;
+use crate::connectors::{Connector, ConnectorBuilder, ConnectorConfig, ConnectorType};
+use crate::errors::Error;
+use googapis::google::api::MonitoredResource;
+use googapis::google::logging::r#type::LogSeverity;
+use serde::Deserialize;
+use simd_json::OwnedValue;
+use std::collections::HashMap;
+use tremor_pipeline::ConfigImpl;
+
+#[derive(Deserialize, Clone)]
+pub(crate) struct Config {
+    // Optional. A default log resource name that is assigned to all log entries in entries that do not specify a value for log_name:
```

Comment:
`///` would turn all those into comments the compiler understands :)

Reply:
Yep. I'll likely rephrase those - they're verbatim from the proto files. I'll replace with doc comments for now